app_customers/lib/main.dart

```diff
@@ -38,6 +39,19 @@ Future<void> main() async {
     service: getAppConfigDefaults().remoteConfigType,
     defaults: getRemoteConfigDefaults(),
   );
+  Logger.root.level = Level.ALL;
+  Logger.root.onRecord.listen((record) {
+    if (kDebugMode && record.level == Level.SEVERE) {
+      print(
```

Comment:
we have to use logger instead of print.

Reply:
the logger listen the message and print using print